pandas/tests/arrays/test_array.py

```diff
@@ -182,6 +182,13 @@ def test_array_copy():
     assert np.shares_memory(a, b._ndarray) is True
 
 
+def test_array_from_series():
```

Comment:
can you instead put in pandas/tests/test_constructors.py

and can you assert the results of each of these with `tm.assert_numpy_array_equal`.